plugins/modules/zos_copy.py

```diff
@@ -563,6 +563,61 @@
     returned: success
     type: str
     sample: SAMPLE.SEQ.DATA.SET
+dest_created:
+    description: Boolean value that indicate the creation of dest.
+    returned: success and if dest is create by the module.
+    type: bool
+    sample: true
+destination_attributes:
+    description: Values of a dest create by the module.
```

Comment:
I think instead of saying "values" you could just say "Attributes", and you could rework it to "Attributes of the destination when created by the module"